custom-connectors/AzureOpenAIService/apiDefinition.swagger.json

```diff
@@ -0,0 +1,657 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure OpenAI Service",
+    "description": "Azure OpenAI Service provides REST API access to OpenAI's powerful language models including the GPT-3, Codex and Embeddings model series. These models can be easily adapted to your specific task including but not limited to content generation, summarization, semantic search, and natural language to code translation.",

```

Comment:
@Laskewitz is it possible to remove the word "API" from the description. Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
Done and fixed.